extensions/replication/ReplicationAPI.js

```diff
@@ -67,23 +69,31 @@ class ReplicationAPI {
      */
     static sendDataMoverAction(producer, action, log, cb) {
         const { bucket, key } = action.getAttribute('target');
-        const kafkaEntries = [{ key: `${bucket}/${key}`,
-                                message: action.toKafkaMessage() }];
-        producer.sendToTopic(dataMoverTopic, kafkaEntries, (err, reports) => {
+        const toLocation = action.getAttribute('toLocation');
+        const locationConfig = locations[toLocation];
+        let topic = dataMoverTopic;
+        if (locationConfig && locationConfig.isCold) {
+            topic = `${constants.coldStorageArchiveTopicPrefix}-${toLocation}`;
```

Comment:
I'd be more comfortable having the `-` part of the prefix, and the prefix in configuration instead of constant.